tests/test-cases/Group3-Docker-Compose/3-03-Docker-Compose-Basic.robot

```diff
@@ -61,6 +71,16 @@ Compose kill
     Log  ${out}
     Should Be Equal As Integers  ${rc}  0
 
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} ps -a
+    Log  ${out}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network inspect vic_default
```

Comment:
Do we need this since we `Check Container Removed From Network` does this anyway?

Reply:
You are right. I'll remove it.